zircon.core/src/main/kotlin/org/hexworks/zircon/internal/component/impl/log/LogElementBuffer.kt

```diff
@@ -0,0 +1,53 @@
+package org.hexworks.zircon.internal.component.impl.log
```

Comment:
I think that the concept of having *row*s is a bit misleading because there are instances where an entry takes up more than one row. Instead of having this buffer we could re-use the functionality the `Container` interface is already giving to us. This is how dropping the first row (because of overflow) would look like:

```
        panel.children.firstOrNull()?.let { 
            it.removeFromParent() // i know this is not there yet
        }
        panel.children.forEach { 
            it.moveUpBy(1)
        }
```